src/_actions/users.actions.js

```diff
@@ -98,21 +98,23 @@ const register = (firstName, lastName, email, telephone, password) => {
   return dispatch => {
     dispatch(request());
 
-    usersService.register(firstName, lastName, email, telephone, password).then(
-      () => {
-        dispatch(success());
-        history.push('/');
-        dispatch(
-          alertActions.success(
-            'Successfully created an account! Please await approval!'
-          )
-        );
-      },
-      () => {
-        dispatch(failure());
-        dispatch(alertActions.error('Failed to create an account!'));
-      }
-    );
+    usersService
+      .register(firstName, lastName, email, telephone, password, token)
+      .then(
+        () => {
+          dispatch(success());
+          history.push('/');
+          dispatch(
+            alertActions.success(
+              'Successfully created an account! Please await approval!'
```

Comment:
Is this still relevant?

Reply:
No, I've just gotten rid of 'Please await approval!'. Will push it now